eng/pipelines/templates/steps/publish-vscode.yml

```diff
@@ -1,28 +1,29 @@
 parameters:
   TagRepository: false
   VsixVersion: $(VSIX_VERSION)
-  UpdateShield: false
   StorageContainerName: '`$web'
   PublishToMarketplace: false
 
 steps:
-  - task: DownloadPipelineArtifact@2
-    inputs:
-      artifact: signed
-      path: signed
-
   - ${{ if eq('true', parameters.TagRepository) }}:
     # Perform tag verification before publishing anything
-    - task: PowerShell@2
-      inputs:
-        pwsh: true
-        targetType: filePath
-        filePath: ./eng/scripts/Set-GitHubReleaseTag.ps1
```

Comment:
I assume you did this because it is happening in the release deployment job. We could still do the tagging in a normal job as well which is what we did in the sdk repos.

Reply:
If we do stick with inline over this script we should probably delete the script.

Reply:
azd uploads signed artifacts when it creates a release. We could upload the artifacts in a separate step but this approach tags, creates the release, and uploads the artifacts in a single job.